python/pyspark/testing/utils.py

```diff
@@ -49,6 +60,20 @@
 SPARK_HOME = _find_spark_home()
 
 
+class bcolors:
```

Comment:
```suggestion
class Bcolors:
```

per PEP8 https://peps.python.org/pep-0008/#class-names